dspace/config/modules/bulkedit.cfg

```diff
@@ -9,7 +9,7 @@
 # bulkedit.valueseparator = ||
 
 # The delimiter used to separate fields (defaults to a comma for CSV)
-# bulkedit.fieldseparator = ,
+bulkedit.fieldseparator = ,
```

Comment:
This is maybe a mistake.

Reply:
That’s correct, it will be removed

Reply:
This should be resolved using commit b37d0e216718316ba202f34556f53344cd0346b8